### PR TITLE
Verification the launching of the tests when LATENCY_TEST_DELAY variable is more than 120 sec 

### PR DESCRIPTION
**Pull request by IlyaTyomkin**
During running the CNF tests there were found the problem what oslat latency test fails when one of Env. variable LATENCY_TEST_DELAY was set with its value more than 120 sec. This PR automate and verify the state when on the test code the value of LATENCY_TEST_DELAY = 200 sec for all latency tests.  All other tests launched with a new value

### CODE REVIEW

File: pkg/operator/status.go [hunk 1]
```diff
@@ -341,6 +341,7 @@ func getRelatedObjects() []configv1.ObjectReference {
 		{Group: "tuned.openshift.io", Resource: "profiles", Name: "", Namespace: tunedMf.Namespace},
 		{Group: "tuned.openshift.io", Resource: "tuneds", Name: "", Namespace: tunedMf.Namespace},
 		{Group: "apps", Resource: "daemonsets", Name: dsMf.Name, Namespace: dsMf.Namespace},
+		{Group: "performance.openshift.io", Resource: "performanceprofiles", Name: ""},
```
Comment (ffromani) [inline pkg/operator/status.go:344]: Is this relevant to this fix?

In addition, the commit message looks confusing, could you please clean it up?
Not sure why @MarSik is listed as author

File: test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go [hunk 2]
```diff
@@ -222,7 +222,10 @@ func getValidValuesTests(toolToTest string) []latencyTest {
 	//testCpus: for tests that expect a success output message, note that an even CPU number is needed, otherwise the test would fail with SMTAlignmentError
 
 	successRuntime := "30"
-	testSet = append(testSet, latencyTest{testDelay: "0", testRun: "true", testRuntime: successRuntime, testMaxLatency: guaranteedLatency, testCpus: "4", outputMsgs: []string{success}, toolToTest: toolToTest})
+
+        testSet = append(testSet, latencyTest{testDelay: "200", testRun: "true", testRuntime: successRuntime, testMaxLatency: guaranteedLatency, testCpus: "4", outputMsgs: []string{success},toolToTest: toolToTest})
```
Comment (ffromani) [inline test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go:226]: missing space between `outputMsgs` and `toolToTest`, this is probably why `gofmt` is complaining.

File: pkg/operator/status.go (hunk 1 above)
Comment (MarSik) [reply pkg/operator/status.go:344]: It looks like a bad rebase, the commit message contains couple of other older commits too

**Comment by IlyaTyomkin** [comment]
PR was closed due to the wrong rebase. The link to new one:
https://github.com/openshift/cluster-node-tuning-operator/pull/662
